presto-docs/src/main/sphinx/release/release-319.rst

```diff
@@ -0,0 +1,45 @@
+===========
+Release 319
+===========
+
+General Changes
+---------------
+* Fix planning failure for queries involving ``UNION`` and ``DISTINCT`` aggregates. (:issue:`1510`)
+* Fix excessive runtime when parsing expressions involving ``CASE``. (:issue:`1407`)
+* Fix fragment's output size in the output of ``EXPLAIN ANALYZE``. (:issue:`1345`)
+* Improve performance of queries against ``information_schema`` tables. (:issue:`1329`, :issue:`1308`)
+* Eliminate cross joins in some queries with coercions and complex expressions
+  on join conditions. (:issue:`1390`)
+* Fix a rare failure when running ``EXPLAIN ANALYZE`` on a query containing
+  window functions. (:issue:`1401`)
+* Include cost estimates in output of ``EXPLAIN (TYPE IO)``. (:issue:`806`)
+* Improve coercion handling for correlated subqueries. (:issue:`1453`)
```

Comment:
What's the user-visible impact of this?

Reply:
if i recall, some queries would fail during planning. @kasiafi?

Reply:
I verified that this causes a failure (in the local execution failure)

Reply:
```
* Fix planning failure due to coercion handling for correlated subqueries. (:issue:`1453`)